docs/source/topic-guides/models.md

```diff
@@ -141,17 +141,17 @@ If there is only one _new owner_, the condition will be a simple signature condi
         },
         "uri": "<string>"
     },
-    "new_owners": ["<new owner public key>"]
+    "owners_after": ["<public key of the owner after the transaction happens>"]
 }
 ```
 
 - **Condition header**:
     - `cid`: Condition index so that we can reference this output as an input to another transaction. It also matches
     the input `fid`, making this the condition to fulfill in order to spend the asset used as input with `fid`.
-    - `new_owners`: A list containing one item: the public key of the new owner.
+    - `owners_after`: A list containing one item: the public key of the owner after the transaction happens.
 - **Condition body**:
     - `bitmask`: A set of bits representing the features required by the condition type.
-    - `public_key`: The _new_owner's_ public key.
+    - `public_key`: The _owner_after's_ public key.
```

Comment:
Haha, I never noticed this one before. It should probably have been left alone, or maybe the italics aren't necessary:

    - `public_key`: The new owner's public key.